js/examples/openai/src/index.ts

```diff
@@ -22,6 +23,23 @@ app.post("/completion", async (req, res) => {
   });
 });
 
+// Streaming
+app.post("/completion/stream", async (req, res) => {
```

Comment:
You can add multiple endpoints ala langserve. Let's add one for each. /completion /stream